python_modules/dagster/dagster/_core/instance/ref.py

```diff
@@ -111,26 +117,38 @@ def configurable_storage_data(
         )
 
     elif "sqlite" in config_field:
-        base_dir = check.str_elem(config_field["sqlite"], "base_dir")
+        base_dir = config_field["sqlite"]["base_dir"]
         storage_data = ConfigurableClassData(
             "dagster._core.storage.sqlite_storage",
             "DagsterSqliteStorage",
             yaml.dump({"base_dir": base_dir}, default_flow_style=False),
         )
-        run_storage_data = ConfigurableClassData(
-            "dagster._core.storage.runs",
-            "SqliteRunStorage",
-            yaml.dump({"base_dir": _runs_directory(base_dir)}, default_flow_style=False),
```

Comment:
this was the main thing that was breaking it - the base dirs are different for the different components so they can't all be sourced from the passed in env var directly